gradle-lite manager does not fetch dependencies from mavenCentral or jcenter when 3rd party repositores are not accessible

**What Renovate type, platform and version are you using?**

* Found on self-hosted Renovate running 24.10.1-slim docker image
* Confirmed on GitHub on 23.12.2020 13:00 UTC

**Describe the bug**
A problem with updating dependencies in `build.gradle` via `gradle-lite` manager.

Renovate updates plugins declared in `build.gradle`. However it does not fetch *dependencies* from Maven Central or JCenter when 3rd party repositories specified in `build.gradle` are not accessible (e.g. wrong url, missing password, **mising hostRule** etc.).

**Relevant debug logs**

<details>
<summary>Debug logs</summary>
<pre>
<code>
INFO: Repository started
{
  "renovateVersion": "24.10.1"
}
DEBUG: Using localDir: /mnt/renovate/gh/BasicWolf/renovate-bugcase-2a
DEBUG: Repository cache not found
{
  "cacheFileName": "/tmp/renovate-cache/renovate/repository/github/BasicWolf/renovate-bugcase-2a.json"
}
DEBUG: initRepo("BasicWolf/renovate-bugcase-2a")
DEBUG: Overriding default GitHub endpoint
{
  "endpoint": "https://api.github.com/"
}
DEBUG: No dangling containers to remove
DEBUG: BasicWolf/renovate-bugcase-2a default branch = master
DEBUG: Using personal access token for git init
DEBUG: resetMemCache()
DEBUG: checkOnboarding()
DEBUG: isOnboarded()
DEBUG: findFile(renovate.json)
DEBUG: Initializing git repository into /mnt/renovate/gh/BasicWolf/renovate-bugcase-2a
DEBUG: git clone completed
{
  "durationMs": 587
}
DEBUG: latest commit
{
  "latestCommitDate": "2020-12-23T14:36:58+02:00"
}
DEBUG: Setting git author name
{
  "gitAuthorName": "Renovate Bot"
}
DEBUG: Setting git author email
{
  "gitAuthorEmail": "bot@renovateapp.com"
}
DEBUG: Config file exists
{
  "fileName": "renovate.json"
}
DEBUG: Retrieving issueList
DEBUG: Retrieved 0 issues
DEBUG: Repo is onboarded
DEBUG: Found renovate.json config file
DEBUG: Repository config
{
  "fileName": "renovate.json",
  "config": {
    "extends": [
      "config:base"
    ],
    "gradle-lite": {
      "enabled": true
    },
    "packageRules": [
      {
        "packagePatterns": [
          "*"
        ],
        "managers": [
          "gradle-lite"
        ],
        "groupName": "Gradle packages"
      }
    ]
  }
}
DEBUG: migrateAndValidate()
DEBUG: No config migration necessary
DEBUG: massaged config
{
  "config": {
    "extends": [
      "github>whitesource/merge-confidence:beta",
      "config:base"
    ],
    "gradle-lite": {
      "enabled": true
    },
    "packageRules": [
      {
        "packagePatterns": [
          "*"
        ],
        "managers": [
          "gradle-lite"
        ],
        "groupName": "Gradle packages"
      }
    ]
  }
}
DEBUG: migrated config
{
  "config": {
    "extends": [
      "github>whitesource/merge-confidence:beta",
      "config:base"
    ],
    "gradle-lite": {
      "enabled": true
    },
    "packageRules": [
      {
        "packagePatterns": [
          "*"
        ],
        "managers": [
          "gradle-lite"
        ],
        "groupName": "Gradle packages"
      }
    ]
  }
}
DEBUG: Found repo ignorePaths
{
  "ignorePaths": [
    "**/node_modules/**",
    "**/bower_components/**",
    "**/vendor/**",
    "**/examples/**",
    "**/__tests__/**",
    "**/test/**",
    "**/tests/**",
    "**/__fixtures__/**"
  ]
}
DEBUG: detectSemanticCommits()
DEBUG: getCommitMessages
DEBUG: Semantic commits detection: unknown
DEBUG: No semantic commits detected
DEBUG: Setting branchPrefix: renovate/
DEBUG: Cannot read vulnerability alerts
DEBUG: No vulnerability alerts found
DEBUG: processRepo()
DEBUG: No baseBranches
DEBUG: extract()
DEBUG: Setting current branch to master
DEBUG: latest commit
{
  "branchName": "master",
  "latestCommitDate": "2020-12-23T14:36:58+02:00"
}
DEBUG: Using file match: (^|/)tasks/[^/]+\.ya?ml$ for manager ansible
DEBUG: Using file match: (^|/)requirements\.ya?ml$ for manager ansible-galaxy
DEBUG: Using file match: azure.*pipelines?.*\.ya?ml$ for manager azure-pipelines
DEBUG: Using file match: (^|/)batect(-bundle)?\.yml$ for manager batect
DEBUG: Using file match: (^|/)batect$ for manager batect-wrapper
DEBUG: Using file match: (^|/)WORKSPACE(|\.bazel)$ for manager bazel
DEBUG: Using file match: \.bzl$ for manager bazel
DEBUG: Using file match: buildkite\.ya?ml for manager buildkite
DEBUG: Using file match: \.buildkite/.+\.ya?ml$ for manager buildkite
DEBUG: Using file match: (^|/)Gemfile$ for manager bundler
DEBUG: Using file match: (^|/)Cargo.toml$ for manager cargo
DEBUG: Using file match: (^|/).circleci/config.yml$ for manager circleci
DEBUG: Using file match: (^|/)Podfile$ for manager cocoapods
DEBUG: Using file match: (^|/)([\w-]*)composer.json$ for manager composer
DEBUG: Using file match: (^|/)deps\.edn$ for manager deps-edn
DEBUG: Using file match: (^|/)docker-compose[^/]*\.ya?ml$ for manager docker-compose
DEBUG: Using file match: (^|/|\.)Dockerfile$ for manager dockerfile
DEBUG: Using file match: (^|/)Dockerfile\.[^/]*$ for manager dockerfile
DEBUG: Using file match: (^|/).drone.yml$ for manager droneci
DEBUG: Using file match: (^|/).gitmodules$ for manager git-submodules
DEBUG: Using file match: ^\.github/workflows/[^/]+\.ya?ml$ for manager github-actions
DEBUG: Using file match: ^\.gitlab-ci\.yml$ for manager gitlabci
DEBUG: Using file match: ^\.gitlab-ci\.yml$ for manager gitlabci-include
DEBUG: Using file match: (^|/)go.mod$ for manager gomod
DEBUG: Using file match: \.gradle(\.kts)?$ for manager gradle
DEBUG: Using file match: (^|/)gradle.properties$ for manager gradle
DEBUG: Using file match: (^|/)gradle.properties$ for manager gradle-lite
DEBUG: Using file match: \.gradle(\.kts)?$ for manager gradle-lite
DEBUG: Using file match: (^|/)gradle/wrapper/gradle-wrapper.properties$ for manager gradle-wrapper
DEBUG: Using file match: (^|/)requirements\.yaml$ for manager helm-requirements
DEBUG: Using file match: (^|/)values.yaml$ for manager helm-values
DEBUG: Using file match: (^|/)helmfile.yaml$ for manager helmfile
DEBUG: Using file match: (^|/)Chart.yaml$ for manager helmv3
DEBUG: Using file match: ^Formula/[^/]+[.]rb$ for manager homebrew
DEBUG: Using file match: \.html?$ for manager html
DEBUG: Using file match: (^|/)plugins\.txt for manager jenkins
DEBUG: Using file match: (^|/)kustomization\.yaml for manager kustomize
DEBUG: Using file match: (^|/)project\.clj$ for manager leiningen
DEBUG: Using file match: \.pom\.xml$ for manager maven
DEBUG: Using file match: (^|/)pom\.xml$ for manager maven
DEBUG: Using file match: (^|/)package.js$ for manager meteor
DEBUG: Using file match: (^|/)mix\.exs$ for manager mix
DEBUG: Using file match: ^.node-version$ for manager nodenv
DEBUG: Using file match: (^|/)package.json$ for manager npm
DEBUG: Using file match: \.(?:cs|fs|vb)proj$ for manager nuget
DEBUG: Using file match: \.(?:props|targets)$ for manager nuget
DEBUG: Using file match: \.config\/dotnet-tools\.json$ for manager nuget
DEBUG: Using file match: (^|/)\.nvmrc$ for manager nvm
DEBUG: Using file match: (^|/)([\w-]*)requirements.(txt|pip)$ for manager pip_requirements
DEBUG: Using file match: (^|/)setup.py$ for manager pip_setup
DEBUG: Using file match: (^|/)Pipfile$ for manager pipenv
DEBUG: Using file match: (^|/)pyproject\.toml$ for manager poetry
DEBUG: Using file match: (^|/)\.pre-commit-config\.yaml$ for manager pre-commit
DEBUG: Using file match: (^|/)pubspec\.ya?ml$ for manager pub
DEBUG: Using file match: (^|/)\.ruby-version$ for manager ruby-version
DEBUG: Using file match: \.sbt$ for manager sbt
DEBUG: Using file match: project/[^/]*.scala$ for manager sbt
DEBUG: Using file match: (^|/)setup\.cfg$ for manager setup-cfg
DEBUG: Using file match: (^|/)Package\.swift for manager swift
DEBUG: Using file match: \.tf$ for manager terraform
DEBUG: Using file match: (^|/)\.terraform-version$ for manager terraform-version
DEBUG: Using file match: (^|/)terragrunt\.hcl$ for manager terragrunt
DEBUG: Using file match: ^.travis.yml$ for manager travis
DEBUG: Matched 3 file(s) for manager gradle: build.gradle, gradle.properties, settings.gradle.kts
DEBUG: Matched 3 file(s) for manager gradle-lite: build.gradle, gradle.properties, settings.gradle.kts
DEBUG: Matched 1 file(s) for manager gradle-wrapper: gradle/wrapper/gradle-wrapper.properties
DEBUG: Extracting dependencies from all gradle files
DEBUG: Creating renovate-plugin.gradle file with renovate gradle plugin
DEBUG: Start gradle command
{
  "cmd": "./gradlew --init-script renovate-plugin.gradle renovate"
}
DEBUG: Using docker to execute
DEBUG: No tag or tagConstraint specified
{
  "image": "docker.io/renovate/gradle"
}
DEBUG: Fetching Docker image: docker.io/renovate/gradle
DEBUG: gradle-wrapper.extractPackageFile()
DEBUG: Gradle Wrapper
{
  "depName": "gradle",
  "currentValue": "6.7",
  "datasource": "gradle-version",
  "versioning": "gradle"
}
DEBUG: Finished fetching Docker image
DEBUG: Executing command
{
  "command": "docker run --rm --name=renovate_gradle --label=renovate_child -v \"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\":\"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\" -v \"/tmp/renovate-cache\":\"/tmp/renovate-cache\" -e GRADLE_OPTS -w \"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\" docker.io/renovate/gradle bash -l -c \"./gradlew --init-script renovate-plugin.gradle renovate\""
}
WARN: Gradle extraction failed
{
  "errMessage": "Command failed: docker run --rm --name=renovate_gradle --label=renovate_child -v \"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\":\"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\" -v \"/tmp/renovate-cache\":\"/tmp/renovate-cache\" -e GRADLE_OPTS -w \"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\" docker.io/renovate/gradle bash -l -c \"./gradlew --init-script renovate-plugin.gradle renovate\"\nError: Could not find or load main class org.gradle.wrapper.GradleWrapperMain\nCaused by: java.lang.ClassNotFoundException: org.gradle.wrapper.GradleWrapperMain\n"
}
DEBUG: Found gradle-lite package files
DEBUG: Found gradle-wrapper package files
DEBUG: Found 4 package file(s)
INFO: Dependency extraction complete
{
  "baseBranch": "master",
  "stats": {
    "managers": {
      "gradle-lite": {
        "fileCount": 3,
        "depCount": 6
      },
      "gradle-wrapper": {
        "fileCount": 1,
        "depCount": 1
      }
    },
    "total": {
      "fileCount": 4,
      "depCount": 7
    }
  }
}
DEBUG: Looking up org.jetbrains.kotlin.jvm:org.jetbrains.kotlin.jvm.gradle.plugin in repository https://plugins.gradle.org/m2/
DEBUG: Looking up io.gitlab.arturbosch.detekt:io.gitlab.arturbosch.detekt.gradle.plugin in repository https://plugins.gradle.org/m2/
DEBUG: Looking up com.tngtech.archunit:archunit-junit5-api in repository https://example.com/
DEBUG: Looking up com.tngtech.archunit:archunit-junit5-engine in repository https://example.com/
DEBUG: Looking up io.gitlab.arturbosch.detekt:detekt-formatting in repository https://example.com/
DEBUG: Found 83 new releases for io.gitlab.arturbosch.detekt:io.gitlab.arturbosch.detekt.gradle.plugin in repository https://plugins.gradle.org/m2/
DEBUG: Content is not found for Maven url: https://example.com/com/tngtech/archunit/archunit-junit5-engine/maven-metadata.xml
DEBUG: Failed to look up dependency com.tngtech.archunit:archunit-junit5-engine (com.tngtech.archunit:archunit-junit5-engine)(packageFile="build.gradle", dependency="com.tngtech.archunit:archunit-junit5-engine")
DEBUG: Content is not found for Maven url: https://example.com/com/tngtech/archunit/archunit-junit5-api/maven-metadata.xml
DEBUG: Failed to look up dependency com.tngtech.archunit:archunit-junit5-api (com.tngtech.archunit:archunit-junit5-api)(packageFile="build.gradle", dependency="com.tngtech.archunit:archunit-junit5-api")
DEBUG: Content is not found for Maven url: https://example.com/io/gitlab/arturbosch/detekt/detekt-formatting/maven-metadata.xml
DEBUG: Failed to look up dependency io.gitlab.arturbosch.detekt:detekt-formatting (io.gitlab.arturbosch.detekt:detekt-formatting)(packageFile="build.gradle", dependency="io.gitlab.arturbosch.detekt:detekt-formatting")
DEBUG: Failed to look up dependency io.gitlab.arturbosch.detekt:detekt-formatting (io.gitlab.arturbosch.detekt:detekt-formatting)(packageFile="build.gradle", dependency="io.gitlab.arturbosch.detekt:detekt-formatting")
DEBUG: Found 58 new releases for org.jetbrains.kotlin.jvm:org.jetbrains.kotlin.jvm.gradle.plugin in repository https://plugins.gradle.org/m2/
DEBUG: Looking up io.gitlab.arturbosch.detekt:io.gitlab.arturbosch.detekt.gradle.plugin in repository https://example.com/
DEBUG: Content is not found for Maven url: https://example.com/io/gitlab/arturbosch/detekt/io.gitlab.arturbosch.detekt.gradle.plugin/maven-metadata.xml
DEBUG: Looking up org.jetbrains.kotlin.jvm:org.jetbrains.kotlin.jvm.gradle.plugin in repository https://example.com/
DEBUG: Content is not found for Maven url: https://example.com/org/jetbrains/kotlin/jvm/org.jetbrains.kotlin.jvm.gradle.plugin/maven-metadata.xml
DEBUG: Package releases lookups complete
{
  "baseBranch": "master"
}
DEBUG: packageFiles with updates
{
  "config": {
    "gradle-lite": [
      {
        "packageFile": "gradle.properties",
        "datasource": "maven",
        "deps": []
      },
      {
        "packageFile": "build.gradle",
        "datasource": "maven",
        "deps": [
          {
            "depType": "plugin",
            "depName": "org.jetbrains.kotlin.jvm",
            "lookupName": "org.jetbrains.kotlin.jvm:org.jetbrains.kotlin.jvm.gradle.plugin",
            "registryUrls": [
              "https://plugins.gradle.org/m2/",
              "https://example.com/"
            ],
            "currentValue": "1.4.20",
            "commitMessageTopic": "plugin org.jetbrains.kotlin.jvm",
            "managerData": {
              "fileReplacePosition": 56,
              "packageFile": "build.gradle"
            },
            "depIndex": 0,
            "updates": [
              {
                "fromVersion": "1.4.20",
                "toVersion": "1.4.21",
                "newValue": "1.4.21",
                "newMajor": 1,
                "newMinor": 4,
                "updateType": "minor",
                "isSingleVersion": true
              }
            ],
            "warnings": [],
            "fixedVersion": "1.4.20",
            "dependencyUrl": "org/jetbrains/kotlin/jvm/org.jetbrains.kotlin.jvm.gradle.plugin"
          },
          {
            "depType": "plugin",
            "depName": "io.gitlab.arturbosch.detekt",
            "lookupName": "io.gitlab.arturbosch.detekt:io.gitlab.arturbosch.detekt.gradle.plugin",
            "registryUrls": [
              "https://plugins.gradle.org/m2/",
              "https://example.com/"
            ],
            "currentValue": "1.14.2",
            "commitMessageTopic": "plugin io.gitlab.arturbosch.detekt",
            "managerData": {
              "fileReplacePosition": 110,
              "packageFile": "build.gradle"
            },
            "depIndex": 1,
            "updates": [
              {
                "fromVersion": "1.14.2",
                "toVersion": "1.15.0",
                "newValue": "1.15.0",
                "newMajor": 1,
                "newMinor": 15,
                "updateType": "minor",
                "isSingleVersion": true
              }
            ],
            "warnings": [],
            "fixedVersion": "1.14.2",
            "dependencyUrl": "io/gitlab/arturbosch/detekt/io.gitlab.arturbosch.detekt.gradle.plugin"
          },
          {
            "depName": "com.tngtech.archunit:archunit-junit5-api",
            "currentValue": "0.14.1",
            "managerData": {
              "fileReplacePosition": 449,
              "packageFile": "build.gradle"
            },
            "registryUrls": [
              "https://example.com/"
            ],
            "depIndex": 2,
            "updates": [],
            "warnings": [
              {
                "depName": "com.tngtech.archunit:archunit-junit5-api",
                "message": "Failed to look up dependency com.tngtech.archunit:archunit-junit5-api"
              }
            ],
            "fixedVersion": "0.14.1"
          },
          {
            "depName": "com.tngtech.archunit:archunit-junit5-engine",
            "currentValue": "0.14.1",
            "managerData": {
              "fileReplacePosition": 550,
              "packageFile": "build.gradle"
            },
            "registryUrls": [
              "https://example.com/"
            ],
            "depIndex": 3,
            "updates": [],
            "warnings": [
              {
                "depName": "com.tngtech.archunit:archunit-junit5-engine",
                "message": "Failed to look up dependency com.tngtech.archunit:archunit-junit5-engine"
              }
            ],
            "fixedVersion": "0.14.1"
          },
          {
            "depName": "io.gitlab.arturbosch.detekt:detekt-formatting",
            "currentValue": "1.14.2",
            "managerData": {
              "fileReplacePosition": 652,
              "packageFile": "build.gradle"
            },
            "registryUrls": [
              "https://example.com/"
            ],
            "depIndex": 4,
            "updates": [],
            "warnings": [
              {
                "depName": "io.gitlab.arturbosch.detekt:detekt-formatting",
                "message": "Failed to look up dependency io.gitlab.arturbosch.detekt:detekt-formatting"
              }
            ],
            "fixedVersion": "1.14.2"
          },
          {
            "depName": "io.gitlab.arturbosch.detekt:detekt-formatting",
            "currentValue": "1.11.0",
            "managerData": {
              "fileReplacePosition": 754,
              "packageFile": "build.gradle"
            },
            "registryUrls": [
              "https://example.com/"
            ],
            "depIndex": 5,
            "updates": [],
            "warnings": [
              {
                "depName": "io.gitlab.arturbosch.detekt:detekt-formatting",
                "message": "Failed to look up dependency io.gitlab.arturbosch.detekt:detekt-formatting"
              }
            ],
            "fixedVersion": "1.11.0"
          }
        ]
      },
      {
        "packageFile": "settings.gradle.kts",
        "datasource": "maven",
        "deps": []
      }
    ],
    "gradle-wrapper": [
      {
        "packageFile": "gradle/wrapper/gradle-wrapper.properties",
        "deps": [
          {
            "depName": "gradle",
            "currentValue": "6.7",
            "datasource": "gradle-version",
            "versioning": "gradle",
            "depIndex": 0,
            "updates": [
              {
                "fromVersion": "6.7",
                "toVersion": "6.7.1",
                "newValue": "6.7.1",
                "newMajor": 6,
                "newMinor": 7,
                "updateType": "minor",
                "isSingleVersion": true,
                "releaseTimestamp": "2020-11-16T17:09:24+0000"
              }
            ],
            "warnings": [],
            "fixedVersion": "6.7",
            "sourceUrl": "https://github.com/gradle/gradle",
            "homepage": "https://gradle.org"
          }
        ]
      }
    ]
  }
}
DEBUG: branchifyUpgrades
DEBUG: Using group branchName template
DEBUG: Dependency org.jetbrains.kotlin.jvm is part of group Gradle packages
DEBUG: Using group branchName template
DEBUG: Dependency io.gitlab.arturbosch.detekt is part of group Gradle packages
DEBUG: 3 flattened updates found: org.jetbrains.kotlin.jvm, io.gitlab.arturbosch.detekt, gradle
DEBUG: Returning 2 branch(es)
DEBUG: config.repoIsOnboarded=true
DEBUG: processRepo()
DEBUG: Processing 2 branches: renovate/gradle-6.x, renovate/gradle-packages
DEBUG: Calculating hourly PRs remaining
DEBUG: Retrieving PR list
DEBUG: Retrieved 0 Pull Requests
DEBUG: currentHourStart=2020-12-23T13:00:00.000+00:00
DEBUG: PR hourly limit remaining: 2
DEBUG: Calculating prConcurrentLimit (20)
DEBUG: getBranchPr(renovate/gradle-packages)
DEBUG: findPr(renovate/gradle-packages, undefined, open)
DEBUG: getBranchPr(renovate/gradle-6.x)
DEBUG: findPr(renovate/gradle-6.x, undefined, open)
DEBUG: 0 PRs are currently open
DEBUG: PR concurrent limit remaining: 20
DEBUG: Calculated maximum PRs remaining this run
{
  "prsRemaining": 2
}
DEBUG: PullRequests limit = 2
DEBUG: getBranchPr(renovate/gradle-packages)(branch="renovate/gradle-packages")
DEBUG: findPr(renovate/gradle-packages, undefined, open)(branch="renovate/gradle-packages")
DEBUG: processBranch with 2 upgrades(branch="renovate/gradle-packages")
DEBUG: Setting current branch to master(branch="renovate/gradle-packages")
DEBUG: latest commit(branch="renovate/gradle-packages")
{
  "branchName": "master",
  "latestCommitDate": "2020-12-23T14:36:58+02:00"
}
DEBUG: getBranchPr(renovate/gradle-packages)(branch="renovate/gradle-packages")
DEBUG: findPr(renovate/gradle-packages, undefined, open)(branch="renovate/gradle-packages")
DEBUG: branchExists=false(branch="renovate/gradle-packages")
DEBUG: Branch has 2 upgrade(s)(branch="renovate/gradle-packages")
DEBUG: recreateClosed is true(branch="renovate/gradle-packages")
DEBUG: Checking schedule(at any time, null)(branch="renovate/gradle-packages")
DEBUG: No schedule defined(branch="renovate/gradle-packages")
DEBUG: Branch needs creating(branch="renovate/gradle-packages")
DEBUG: Using reuseExistingBranch: false(branch="renovate/gradle-packages")
DEBUG: manager.getUpdatedPackageFiles()(branch="renovate/gradle-packages")
{
  "reuseExistingBranch": false,
  "branchName": "renovate/gradle-packages"
}
DEBUG: Updating packageFile content(packageFile="build.gradle", branch="renovate/gradle-packages")
{
  "depName": "io.gitlab.arturbosch.detekt"
}
DEBUG: Updating packageFile content(packageFile="build.gradle", branch="renovate/gradle-packages")
{
  "depName": "org.jetbrains.kotlin.jvm"
}
DEBUG: Updated 1 package files(branch="renovate/gradle-packages")
DEBUG: No updated lock files in branch(branch="renovate/gradle-packages")
DEBUG: 1 file(s) to commit(branch="renovate/gradle-packages")
DEBUG: Committing files to branch renovate/gradle-packages(branch="renovate/gradle-packages")
INFO: Branch created(branch="renovate/gradle-packages")
{
  "commitSha": "5fc7ca6"
}
DEBUG: Ensuring PR(branch="renovate/gradle-packages")
DEBUG: There are 0 errors and 0 warnings(branch="renovate/gradle-packages")
DEBUG: getBranchPr(renovate/gradle-packages)(branch="renovate/gradle-packages")
DEBUG: findPr(renovate/gradle-packages, undefined, open)(branch="renovate/gradle-packages")
DEBUG: Creating PR(branch="renovate/gradle-packages")
{
  "prTitle": "Update Gradle packages"
}
DEBUG: Creating PR(branch="renovate/gradle-packages")
{
  "title": "Update Gradle packages",
  "head": "BasicWolf:renovate/gradle-packages",
  "base": "master",
  "draft": false
}
DEBUG: PR created(branch="renovate/gradle-packages")
{
  "pr": 1,
  "draft": false
}
DEBUG: Adding labels '' to #1(branch="renovate/gradle-packages")
INFO: PR created(branch="renovate/gradle-packages")
{
  "pr": 1,
  "prTitle": "Update Gradle packages"
}
DEBUG: Created Pull Request #1(branch="renovate/gradle-packages")
DEBUG: Checking #1 for automerge(branch="renovate/gradle-packages")
{
  "automerge": false,
  "automergeType": "pr",
  "automergeComment": "automergeComment"
}
DEBUG: No automerge(branch="renovate/gradle-packages")
DEBUG: getBranchPr(renovate/gradle-6.x)(branch="renovate/gradle-6.x")
DEBUG: findPr(renovate/gradle-6.x, undefined, open)(branch="renovate/gradle-6.x")
DEBUG: processBranch with 1 upgrades(branch="renovate/gradle-6.x")
DEBUG: Setting current branch to master(branch="renovate/gradle-6.x")
DEBUG: latest commit(branch="renovate/gradle-6.x")
{
  "branchName": "master",
  "latestCommitDate": "2020-12-23T14:36:58+02:00"
}
DEBUG: getBranchPr(renovate/gradle-6.x)(branch="renovate/gradle-6.x")
DEBUG: findPr(renovate/gradle-6.x, undefined, open)(branch="renovate/gradle-6.x")
DEBUG: branchExists=false(branch="renovate/gradle-6.x")
DEBUG: Branch has 1 upgrade(s)(branch="renovate/gradle-6.x")
DEBUG: recreateClosed is false(branch="renovate/gradle-6.x")
DEBUG: findPr(renovate/gradle-6.x, Update dependency gradle to v6.7.1, !open)(branch="renovate/gradle-6.x")
DEBUG: prAlreadyExisted=false(branch="renovate/gradle-6.x")
DEBUG: Checking schedule(at any time, null)(branch="renovate/gradle-6.x")
DEBUG: No schedule defined(branch="renovate/gradle-6.x")
DEBUG: Branch needs creating(branch="renovate/gradle-6.x")
DEBUG: Using reuseExistingBranch: false(branch="renovate/gradle-6.x")
DEBUG: manager.getUpdatedPackageFiles()(branch="renovate/gradle-6.x")
{
  "reuseExistingBranch": false,
  "branchName": "renovate/gradle-6.x"
}
DEBUG: Starting search at index 131(packageFile="gradle/wrapper/gradle-wrapper.properties", branch="renovate/gradle-6.x")
{
  "depName": "gradle"
}
DEBUG: Found match at index 131(packageFile="gradle/wrapper/gradle-wrapper.properties", branch="renovate/gradle-6.x")
{
  "depName": "gradle"
}
DEBUG: gradle-wrapper.extractPackageFile()(branch="renovate/gradle-6.x")
DEBUG: Gradle Wrapper(branch="renovate/gradle-6.x")
{
  "depName": "gradle",
  "currentValue": "6.7.1",
  "datasource": "gradle-version",
  "versioning": "gradle"
}
DEBUG: Contents updated(packageFile="gradle/wrapper/gradle-wrapper.properties", branch="renovate/gradle-6.x")
{
  "depName": "gradle"
}
DEBUG: gradle-wrapper.updateArtifacts()(branch="renovate/gradle-6.x")
{
  "updatedDeps": [
    "gradle"
  ]
}
DEBUG: Updating gradle wrapper: "./gradlew wrapper --gradle-distribution-url https://services.gradle.org/distributions/gradle-6.7.1-bin.zip"(branch="renovate/gradle-6.x")
DEBUG: Using docker to execute(branch="renovate/gradle-6.x")
DEBUG: No tag or tagConstraint specified(branch="renovate/gradle-6.x")
{
  "image": "docker.io/renovate/gradle"
}
DEBUG: Docker image is already prefetched: docker.io/renovate/gradle(branch="renovate/gradle-6.x")
DEBUG: Executing command(branch="renovate/gradle-6.x")
{
  "command": "docker run --rm --name=renovate_gradle --label=renovate_child -v \"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\":\"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\" -v \"/tmp/renovate-cache\":\"/tmp/renovate-cache\" -e GRADLE_OPTS -w \"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\" docker.io/renovate/gradle bash -l -c \"./gradlew wrapper --gradle-distribution-url https://services.gradle.org/distributions/gradle-6.7.1-bin.zip\""
}
WARN: Error executing gradle wrapper update command. It can be not a critical one though.(branch="renovate/gradle-6.x")
{
  "err": {
    "killed": false,
    "code": 1,
    "signal": null,
    "cmd": "docker run --rm --name=renovate_gradle --label=renovate_child -v \"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\":\"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\" -v \"/tmp/renovate-cache\":\"/tmp/renovate-cache\" -e GRADLE_OPTS -w \"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\" docker.io/renovate/gradle bash -l -c \"./gradlew wrapper --gradle-distribution-url https://services.gradle.org/distributions/gradle-6.7.1-bin.zip\"",
    "stdout": "",
    "stderr": "Error: Could not find or load main class org.gradle.wrapper.GradleWrapperMain\nCaused by: java.lang.ClassNotFoundException: org.gradle.wrapper.GradleWrapperMain\n",
    "message": "Command failed: docker run --rm --name=renovate_gradle --label=renovate_child -v \"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\":\"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\" -v \"/tmp/renovate-cache\":\"/tmp/renovate-cache\" -e GRADLE_OPTS -w \"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\" docker.io/renovate/gradle bash -l -c \"./gradlew wrapper --gradle-distribution-url https://services.gradle.org/distributions/gradle-6.7.1-bin.zip\"\nError: Could not find or load main class org.gradle.wrapper.GradleWrapperMain\nCaused by: java.lang.ClassNotFoundException: org.gradle.wrapper.GradleWrapperMain\n",
    "stack": "Error: Command failed: docker run --rm --name=renovate_gradle --label=renovate_child -v \"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\":\"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\" -v \"/tmp/renovate-cache\":\"/tmp/renovate-cache\" -e GRADLE_OPTS -w \"/mnt/renovate/gh/BasicWolf/renovate-bugcase-2a\" docker.io/renovate/gradle bash -l -c \"./gradlew wrapper --gradle-distribution-url https://services.gradle.org/distributions/gradle-6.7.1-bin.zip\"\nError: Could not find or load main class org.gradle.wrapper.GradleWrapperMain\nCaused by: java.lang.ClassNotFoundException: org.gradle.wrapper.GradleWrapperMain\n\n    at ChildProcess.exithandler (child_process.js:308:12)\n    at ChildProcess.emit (events.js:314:20)\n    at ChildProcess.EventEmitter.emit (domain.js:483:12)\n    at maybeClose (internal/child_process.js:1021:16)\n    at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)"
  }
}
DEBUG: Returning updated gradle-wrapper files(branch="renovate/gradle-6.x")
{
  "files": [
    "gradle/wrapper/gradle-wrapper.properties"
  ]
}
DEBUG: Updated 1 package files(branch="renovate/gradle-6.x")
DEBUG: Updated 1 lock files(branch="renovate/gradle-6.x")
{
  "updatedArtifacts": [
    "gradle/wrapper/gradle-wrapper.properties"
  ]
}
DEBUG: 1 file(s) to commit(branch="renovate/gradle-6.x")
DEBUG: Committing files to branch renovate/gradle-6.x(branch="renovate/gradle-6.x")
INFO: Branch created(branch="renovate/gradle-6.x")
{
  "commitSha": "dc9f065"
}
DEBUG: Ensuring PR(branch="renovate/gradle-6.x")
DEBUG: There are 0 errors and 0 warnings(branch="renovate/gradle-6.x")
DEBUG: getBranchPr(renovate/gradle-6.x)(branch="renovate/gradle-6.x")
DEBUG: findPr(renovate/gradle-6.x, undefined, open)(branch="renovate/gradle-6.x")
DEBUG: Creating PR(branch="renovate/gradle-6.x")
{
  "prTitle": "Update dependency gradle to v6.7.1"
}
DEBUG: Creating PR(branch="renovate/gradle-6.x")
{
  "title": "Update dependency gradle to v6.7.1",
  "head": "BasicWolf:renovate/gradle-6.x",
  "base": "master",
  "draft": false
}
DEBUG: PR created(branch="renovate/gradle-6.x")
{
  "pr": 2,
  "draft": false
}
DEBUG: Adding labels '' to #2(branch="renovate/gradle-6.x")
INFO: PR created(branch="renovate/gradle-6.x")
{
  "pr": 2,
  "prTitle": "Update dependency gradle to v6.7.1"
}
DEBUG: Created Pull Request #2(branch="renovate/gradle-6.x")
DEBUG: No graphql data, returning empty list(branch="renovate/gradle-6.x")
{
  "query": "\n      query {\n        repository(owner: \"BasicWolf\", name: \"renovate-bugcase-2a\") {\n          pullRequests(states: [CLOSED, MERGED], first: 100, orderBy: {field: UPDATED_AT, direction: DESC}) {\n            nodes {\n              number\n              state\n              headRefName\n              title\n              comments(last: 100) {\n                nodes {\n                  databaseId\n                  body\n                }\n              }\n            }\n          }\n        }\n      }\n      "
}
DEBUG: Getting comments for #2(branch="renovate/gradle-6.x")
DEBUG: Found 0 comments(branch="renovate/gradle-6.x")
DEBUG: Checking #2 for automerge(branch="renovate/gradle-6.x")
{
  "automerge": false,
  "automergeType": "pr",
  "automergeComment": "automergeComment"
}
DEBUG: No automerge(branch="renovate/gradle-6.x")
DEBUG: getBranchPr(renovate/gradle-packages)
DEBUG: findPr(renovate/gradle-packages, undefined, open)
DEBUG: Found PR #1
DEBUG: Returning from graphql open PR list
DEBUG: getBranchPr(renovate/gradle-6.x)
DEBUG: findPr(renovate/gradle-6.x, undefined, open)
DEBUG: Found PR #2
DEBUG: Returning from graphql open PR list
DEBUG: Removing any stale branches
DEBUG: config.repoIsOnboarded=true
DEBUG: Branch lists
{
  "branchList": [
    "renovate/gradle-packages",
    "renovate/gradle-6.x"
  ],
  "renovateBranches": [
    "renovate/gradle-packages",
    "renovate/gradle-6.x"
  ]
}
DEBUG: remainingBranches=
DEBUG: No branches to clean up
DEBUG: Repository timing splits (milliseconds)
{
  "splits": {
    "init": 2612,
    "extract": 4890,
    "lookup": 806,
    "update": 15103
  },
  "total": 23926
}
DEBUG: http statistics
{
  "hostStats": [
    "api.github.com, 10 requests, 437ms request average, 0ms queue average",
    "plugins.gradle.org, 3 requests, 292ms request average, 0ms queue average",
    "services.gradle.org, 1 request, 68ms request average, 0ms queue average"
  ],
  "totalRequests": 14
}
INFO: Repository finished
{
  "durationMs": 23926
}
</pre>
</code>
</details>

**Have you created a minimal reproduction repository?**

https://github.com/BasicWolf/renovate-bugcase-2a/

The pull request has `id 'io.gitlab.arturbosch.detekt'` updated to version `'1.15.0'`. However the corresponding dependency in `dependencies { ... }` section remains at old version:
```
dependencies {
    ...
    detektPlugins group: 'io.gitlab.arturbosch.detekt', name: 'detekt-formatting', version: '1.14.2'
}
```

--- a/lib/manager/gradle-lite/common.ts
+++ b/lib/manager/gradle-lite/common.ts
@@ -1,5 +1,9 @@
 import { PackageDependency } from '../common';
 
+export { MAVEN_REPO } from '../../datasource/maven/common';
+
+export const JCENTER_REPO = 'https://jcenter.bintray.com/';
+
 export interface ManagerData {
   fileReplacePosition: number;
   packageFile?: string;

--- a/lib/manager/gradle-lite/common.ts
+++ b/lib/manager/gradle-lite/common.ts
@@ -3,6 +3,7 @@ import { PackageDependency } from '../common';
 export { MAVEN_REPO } from '../../datasource/maven/common';
 
 export const JCENTER_REPO = 'https://jcenter.bintray.com/';
+export const GOOGLE_REPO = 'https://dl.google.com/android/maven2/';
 
 export interface ManagerData {
   fileReplacePosition: number;

--- a/lib/manager/gradle-lite/parser.spec.ts
+++ b/lib/manager/gradle-lite/parser.spec.ts
@@ -1,6 +1,6 @@
 import { readFileSync } from 'fs';
 import path from 'path';
-import { JCENTER_REPO, MAVEN_REPO } from './common';
+import { GOOGLE_REPO, JCENTER_REPO, MAVEN_REPO } from './common';
 import { parseGradle, parseProps } from './parser';
 
 function getGradleFile(fileName: string): string {
@@ -47,12 +47,13 @@ describe('manager/gradle-lite/parser', () => {
     expect(urls).toStrictEqual(['https://example.com']);
 
     ({ urls } = parseGradle(
-      'mavenCentral(); uri("https://example.com"); jcenter();'
+      'mavenCentral(); uri("https://example.com"); jcenter(); google();'
     ));
     expect(urls).toStrictEqual([
       MAVEN_REPO,
       'https://example.com',
       JCENTER_REPO,
+      GOOGLE_REPO,
     ]);
   });
   it('parses long form deps', () => {

--- a/lib/manager/gradle-lite/parser.spec.ts
+++ b/lib/manager/gradle-lite/parser.spec.ts
@@ -1,5 +1,6 @@
 import { readFileSync } from 'fs';
 import path from 'path';
+import { JCENTER_REPO, MAVEN_REPO } from './common';
 import { parseGradle, parseProps } from './parser';
 
 function getGradleFile(fileName: string): string {
@@ -45,8 +46,14 @@ describe('manager/gradle-lite/parser', () => {
     ({ urls } = parseGradle('uri "https://example.com"'));
     expect(urls).toStrictEqual(['https://example.com']);
 
-    ({ urls } = parseGradle('uri("https://example.com")'));
-    expect(urls).toStrictEqual(['https://example.com']);
+    ({ urls } = parseGradle(
+      'mavenCentral(); uri("https://example.com"); jcenter();'
+    ));
+    expect(urls).toStrictEqual([
+      MAVEN_REPO,
+      'https://example.com',
+      JCENTER_REPO,
+    ]);
   });
   it('parses long form deps', () => {
     let deps;

--- a/lib/manager/gradle-lite/parser.ts
+++ b/lib/manager/gradle-lite/parser.ts
@@ -4,6 +4,7 @@ import { logger } from '../../logger';
 import { regEx } from '../../util/regex';
 import { PackageDependency } from '../common';
 import {
+  GOOGLE_REPO,
   JCENTER_REPO,
   MAVEN_REPO,
   ManagerData,
@@ -190,7 +191,7 @@ function processPlugin({
   return { deps: [dep] };
 }
 
-function processRegistryUrl({
+function processCustomRegistryUrl({
   tokenMap,
 }: SyntaxHandlerInput): SyntaxHandlerOutput {
   const registryUrl = tokenMap.registryUrl?.value;
@@ -205,6 +206,18 @@ function processRegistryUrl({
     // no-op
   }
   return null;
+}
+
+function processPredefinedRegistryUrl({
+  tokenMap,
+}: SyntaxHandlerInput): SyntaxHandlerOutput {
+  const registryName = tokenMap.registryName?.value;
+  const registryUrl = {
+    mavenCentral: MAVEN_REPO,
+    jcenter: JCENTER_REPO,
+    google: GOOGLE_REPO,
+  }[registryName];
+  return { urls: [registryUrl] };
 }
 
 function processLongFormDep({
@@ -301,22 +314,16 @@ const matcherConfigs: SyntaxMatchConfig[] = [
   {
     // mavenCentral()
     matchers: [
-      { matchType: TokenType.Word, matchValue: 'mavenCentral' },
+      {
+        matchType: TokenType.Word,
+        matchValue: ['mavenCentral', 'jcenter', 'google'],
+        tokenMapKey: 'registryName',
+      },
       { matchType: TokenType.LeftParen },
       { matchType: TokenType.RightParen },
       endOfInstruction,
     ],
-    handler: () => ({ urls: [MAVEN_REPO] }),
-  },
-  {
-    // jcenter()
-    matchers: [
-      { matchType: TokenType.Word, matchValue: 'jcenter' },
-      { matchType: TokenType.LeftParen },
-      { matchType: TokenType.RightParen },
-      endOfInstruction,
-    ],
-    handler: () => ({ urls: [JCENTER_REPO] }),
+    handler: processPredefinedRegistryUrl,
   },
   {
     // url 'https://repo.spring.io/snapshot/'
@@ -325,7 +332,7 @@ const matcherConfigs: SyntaxMatchConfig[] = [
       { matchType: TokenType.String, tokenMapKey: 'registryUrl' },
       endOfInstruction,
     ],
-    handler: processRegistryUrl,
+    handler: processCustomRegistryUrl,
   },
   {
     // url('https://repo.spring.io/snapshot/')
@@ -336,7 +343,7 @@ const matcherConfigs: SyntaxMatchConfig[] = [
       { matchType: TokenType.RightParen },
       endOfInstruction,
     ],
-    handler: processRegistryUrl,
+    handler: processCustomRegistryUrl,
   },
   {
     // group: "com.example", name: "my.dependency", version: "1.2.3"

--- a/lib/manager/gradle-lite/parser.ts
+++ b/lib/manager/gradle-lite/parser.ts
@@ -4,6 +4,8 @@ import { logger } from '../../logger';
 import { regEx } from '../../util/regex';
 import { PackageDependency } from '../common';
 import {
+  JCENTER_REPO,
+  MAVEN_REPO,
   ManagerData,
   PackageVariables,
   StringInterpolation,
@@ -295,6 +297,26 @@ const matcherConfigs: SyntaxMatchConfig[] = [
       endOfInstruction,
     ],
     handler: processPlugin,
+  },
+  {
+    // mavenCentral()
+    matchers: [
+      { matchType: TokenType.Word, matchValue: 'mavenCentral' },
+      { matchType: TokenType.LeftParen },
+      { matchType: TokenType.RightParen },
+      endOfInstruction,
+    ],
+    handler: () => ({ urls: [MAVEN_REPO] }),
+  },
+  {
+    // jcenter()
+    matchers: [
+      { matchType: TokenType.Word, matchValue: 'jcenter' },
+      { matchType: TokenType.LeftParen },
+      { matchType: TokenType.RightParen },
+      endOfInstruction,
+    ],
+    handler: () => ({ urls: [JCENTER_REPO] }),
   },
   {
     // url 'https://repo.spring.io/snapshot/'

